### DISCUSSION TITLE
Todo comments

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#3939] [General]
``` lua
  { "<leader>xT", "<cmd>Trouble todo toggle filter = {tag = {TODO,FIX,FIXME}}<cr>", desc = "Todo/Fix/Fixme (Trouble)" },
    { "<leader>sT", "<cmd>TodoTelescope keywords=TODO,FIX,FIXME<cr>", desc = "Todo/Fix/Fixme" }
```
the tags , todo,fix and fixme are hard coded on todo comments spec.
Is there a way to supply user tags without having to override these keybinds

**Comment by folke** [comment]
just override the bindings? Why would that be bad?

**Reply by rubiin** [reply]
when switching from telescope to fzf and vice versa I will also have to update the corresponding command `<cmd> TodoTelscope` but i think lazy handles this internally
